docs/getting_started.md

```diff
@@ -18,9 +18,11 @@ chmod +x /usr/local/bin/argocd
 
 ## 2. Install ArgoCD
 ```
-argocd install
+kubectl create namespace argocd
+kubectl apply -n argocd -f https://github.com/argoproj/argo-cd/blob/master/manifests/install.yaml
 ```
-This will create a new namespace, `argocd`, where ArgoCD services and application resources will live.
+This will create a new namespace, `argocd` where ArgoCD services and application resources will live.
+This will also create key replicasets, deployments and services needed for ArgoCD
```

Comment:
It looks like we're missing a step between step 2 and 3. We need instructions on how to login to ArgoCD using the pod name as the password.

Also, it seems like we should have a API to reset password. I'll file another issue for this. I think it's important for v0.6